### PR TITLE
Add plot mirroring back in

### PR DESCRIPTION
**Pull request by frankduncan**
@SethTisue This one is a bit larger.  Any comments you have are welcome, but if it's something you're not interested in reviewing, that's ok too.

[Review on Reviewable](https://reviewable.io/reviews/netlogo/netlogo/735)

### CODE REVIEW

**Comment by SethTisue** [comment]
I gave it a 60-second once-over, looks fine. I assume you just reverted e978d15368f2ea076235191a1c6797ee53bcd147 and did fixups? If it compiles and if it appears to work in light testing, then it meets the same standard it ever met.

**Comment by SethTisue** [comment]
what about the user manual changes though? is it intentional that the feature is now undocumented?

**Comment by frankduncan** [comment]
Oh good call, I'll take care of that today.  And yeah, revert and testing of models uri sent me that use the feature all seemed to work well.

File: dist/i18n/GUI_Strings_ru.txt [hunk 1]
```diff
@@ -31,6 +31,7 @@ menu.file.open = Открыть...
 menu.file.modelsLibrary = Библиотека моделей
 menu.file.save = Сохранить
 menu.file.saveAs = Сохранить как...
+menu.file.saveClientAsApplet = Сохранить клиент как апплет (deprecated)...
```
Comment (SethTisue) [inline dist/i18n/GUI_Strings_ru.txt]: I am demoing GitHub for some coworkers, ignore this comment

**Pull request merged**